src/WebformCivicrmBase.php

```diff
@@ -365,42 +365,42 @@ protected function reorderByLocationType($c, $ent, $values = []){
    * @param array $values
    * @return array $reorderedArray
    */
-  protected function matchLocationTypes($c, $ent, &$values){
+  protected function matchLocationTypes($c, $ent, &$values) {
     // create temporary settings array to include 'user-select' fields
     // on the right place in array
     $settingsArray = $this->add_user_select_field_placeholder($ent, $this->settings['data']['contact'][$c]);
     $userSelectIndex = 0;
     // Go through the array and match up locations by type
     // Put placeholder 'user-select' where location_type_id is empty for second pass
     foreach ($settingsArray[$ent] as $setting) {
-      $valueFound = false;
-      foreach($values as $key => $value) {
+      $valueFound = TRUE;
```

Comment:
@stesi561 `$valueFound` assignment is reversed here. was this intentional?

Reply:
No. No. No. it wasn't. !!! :(